sxt/field12/operation/cmov.cc

```diff
@@ -28,4 +28,14 @@ CUDA_CALLABLE void cmov(f12t::element& f, const f12t::element& g, unsigned int b
     f[i] = f[i] ^ (mask & (f[i] ^ g[i]));
   }
 }
+
+//--------------------------------------------------------------------------------------------------
+// cmov
+//--------------------------------------------------------------------------------------------------
+CUDA_CALLABLE void cmov(uint8_t& f, const uint8_t g, unsigned int b) noexcept {
```

Comment:
Please move this version of cmov into the base/num package.

Code that isn't field-specific and could potentially have reuse in other contexts should be in a more basic package.

Reply:
Great idea. Moved and made a template accepting only integral types.

Reply:
> The cmov component in the base field, field12, can now move uint8_t values

Please update description.

Reply:
Good catch, updated.